python-backend/app/api/mines/namespace.py

```diff
@@ -61,6 +60,10 @@
 api.add_resource(MineSubscriptionResource, '/<string:mine_guid>/subscribe')
 api.add_resource(MineSubscriptionListResource, '/subscribe')
 
+api.add_resource(ApplicationListResource, '/<string:mine_guid>/applications')
+api.add_resource(ApplicationResource, '/<string:mine_guid>/applications/<string:application_guid>')
+api.add_resource(ApplicationStatusCodeResource, '/applications/status-codes')
```

Comment:
Along with what Jon mentioned, I'm concerned about the API endpoint changes here. Since I don't see the accompanying changes in the FE

Reply:
Now they have been added and Paul has identified the constants.